### PR TITLE
utc capi: Added linear gradient tests.

### PR DESCRIPTION
**Pull request by mihashco**

### CODE REVIEW

File: test/capi/testLinearGradient.cpp [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include <stdint.h>
```
Comment (hermet) [inline test/capi/testLinearGradient.cpp:23]: -- unnecessary

File: test/capi/testLinearGradient.cpp [hunk 2]
```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2020-2021 Samsung Electronics Co., Ltd. All rights reserved.
```
Comment (hermet) [inline test/capi/testLinearGradient.cpp:2]: 2021

File: test/capi/testLinearGradient.cpp [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include <stdint.h>
+#include <thorvg_capi.h>
+#include "catch.hpp"
+
+TEST_CASE("Basic Create", "[capi-radial-gradient]")
```
Comment (hermet) [inline test/capi/testLinearGradient.cpp:27]: please use naming.
capiRadialGradient

**Review by hermet — CHANGES_REQUESTED** [review]

File: test/capi/testLinearGradient.cpp [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+TEST_CASE("Set gradient start and end position", "[capi-radial-gradient]")
+{
+    Tvg_Gradient *gradient = NULL;
+    float x1 = 0.0, y1 = 0.0, x2 = 0.0, y2 = 0.0;
+
+    gradient = tvg_linear_gradient_new();
+
+    REQUIRE(gradient != NULL);
+    REQUIRE(tvg_linear_gradient_set(gradient, 10.0, 10.0, 50.0, 50.0) == TVG_RESULT_SUCCESS);
+
```
Comment (mgrudzinska) [inline test/capi/testLinearGradient.cpp:44]: I would use different values for each arg so we could catch mistakes like switching the variables (ex x1 with y1 etc)

**Pull request merged**